lib/taiko.js

```diff
@@ -553,13 +573,24 @@ module.exports.openTab = async (
  * await closeTab('Open Source Test Automation Framework | Gauge')
  * # Closes all the tabs with URL 'https://gauge.org'.
  * await closeTab('https://gauge.org')
+ * # Closes all the tabs with Regex Title 'Go*gle'
+ * await closeTab(/Go*gle/)
+ * # Closes all the tabs with Regex URL '/http(s?):\/\/(www?).google.(com|co.in|co.uk)/'
+ * await closeTab(/http(s?):\/\/(www?).google.(com|co.in|co.uk)/)
  *
  * @param {string} [targetUrl=undefined] - URL/Page title of the tab to close.
  *
  * @returns {Promise}
  */
 module.exports.closeTab = async targetUrl => {
   validate();
+  if (
+    targetUrl != null &&
```

Comment:
@zabil  Why is the null check important here? Because if we don't check it, then it will parse it into regex (/*/) and this will match to all the window , hence closing all the tabs & the window.